sql-statements/sql-statement-change-drainer.md

```diff
@@ -0,0 +1,71 @@
+---
+title: CHANGE DRAINER
+summary: An overview of the usage of CHANGE DRAINER for the TiDB database.
+category: reference
+---
+
+# CHANGE DRAINER
+
+The `CHANGE DRAINER` statement modifies the status information for Drainer in the cluster.
+
+> **Tip:**
+>
+> Drainer's state is automatically reported to PD during running, and the statement is used to modify the state information stored in PD only when the state under abnormal circumstances to be inconsistent with the state information stored in PD.
+
+## Examples
+
+{{< copyable "sql" >}}
+
+```sql
+SHOW DRAINER STATUS;
+```
+
+```sql
++----------|----------------|--------|--------------------|---------------------|
+|  NodeID  |     Address    | State  |   Max_Commit_Ts    |    Update_Time      |
++----------|----------------|--------|--------------------|---------------------|
+| drainer1 | 127.0.0.3:8249 | Online | 408553768673342532 | 2019-04-30 00:00:03 |
++----------|----------------|--------|--------------------|---------------------|
+| drainer2 | 127.0.0.4:8249 | Online | 408553768673345531 | 2019-05-01 00:00:04 |
++----------|----------------|--------|--------------------|---------------------|
+2 rows in set (0.00 sec)
+```
+
+It can be seen that drainer1's state has not been updated for more than a day, the Drainer is in an abnormal state, but the `State` remains' Online '. After using 'CHANGE DRAINER', the Drainer's `State` is changed to 'paused':
```

Comment:
```suggestion
It can be seen that drainer1's state has not been updated for more than a day, the Drainer is in an abnormal state, but the `State` remains `Online`. After using `CHANGE DRAINER`, the Drainer's `State` is changed to 'paused':
```